### ISSUE TITLE
Improve training data creation

### ISSUE THREAD

**Issue by skekre98** [skekre98/NBA-Search#26]
**Description of feature/enhancement**
Currently the training set being created makes all words lower case. This will affect how the model inferences queries negatively. Recreate training set so player names are upper case.

**Description of implementation**
In **preprocess.py**, you will see two functions called `generate_rank_queries` and `generate_stat_queries`. Both of these functions call the `funnel_name` function which is setting all names to lower case. Changes this function to randomly choose a name from _alltime_player_list_ in `../data/text_data.py` without setting it to lower case and regenerate the training data. You can overwrite the data currently in query.csv.

**Why this is a good feature/enhancement**
This will improve query classification by making training data more similar to true data.

**Additional context**
I will also change the class labels from 1 and 2 to 'rank' and 'stat'

**Comment by skekre98** [comment]
Class labels have been converted

**Comment by trynnaBeADev** [comment]
hey  @skekre98 , Is this still open for grab ? If yes , Can I be assigned.

**Comment by skekre98** [comment]
@trynnaBeADev Yes, that would be very helpful, thanks!
